src/routes/guides/building-a-data-source/+page.svelte

```diff
@@ -23,346 +23,112 @@
-    and querying your database or other systems. The original implementation required an
-    <em> API_KEY</em>
-    and an <em>API_ENDPOINT</em>, which our data source does not require, so we can remove those
-    from <em>settings.py</em>. Those can however be handy examples for your own implementation.
-  </p>
-  <SectionTitle title="Adding the route" />
-  <p>
-    Let's open the file <em>app/routers/dataproduct.py</em>. We start with emptying it, so we get
-    rid of the old route. Let's add our new route like this:
-  </p>
-  <Code lang={python}>
-    {`
-from fastapi import APIRouter
-from fastapi.exceptions import HTTPException
-
-from app.datasource import get_data
-from app.models import BasicCountryInfoRequest, BasicCountryInfoResponse
-
-router = APIRouter()
-
-
-@router.post(
-    "/test/ioxio-dataspace-guides/Country/BasicInfo",
-    summary="Basic Country Info",
-    description="Data Product for basic country info",
-    response_model=BasicCountryInfoResponse,
-)
-async def data_product(params: BasicCountryInfoRequest):
-    try:
-        data = await get_data(params.code)
-    except KeyError:
-        raise HTTPException(404, "Country not found")
-
-    return BasicCountryInfoResponse(**data)
-`}
-  </Code>
-  <p>
-    We define the route for the path <em>/test/ioxio-dataspace-guides/Country/BasicInfo</em>, which
-    matches the path for our definition and the path defined in the OpenAPI spec file. We also
-    define which pydantic models we use for the request and for the response, as well as define some
-    metadata for the route. We use our <em>get_data</em> function to retrieve the data, raising a 404
-    exception in case the country is not found.
-  </p>
-  <p>
-    Finally we wrap the data into a <em>BasicCountryInfoResponse</em>. We could as well just return
-    the data as a dictionary and let FastAPI take care of the rest automatically using the
-    definition in the
-    <em>response_model</em>. The
-    <A href="https://fastapi.tiangolo.com/learn/">FastAPI documentation</A>
-    is really well written and describes in great detail topics, such as the
-    <A href="https://fastapi.tiangolo.com/tutorial/body/">request body</A>,
-    <A href="https://fastapi.tiangolo.com/tutorial/response-model/">response models</A>
-    and
-    <A href="https://fastapi.tiangolo.com/tutorial/handling-errors/">handling errors</A>, so those
-    resources are well worth a look if the explanation in this guide was too brief.
-  </p>
-  <SectionTitle title="Deploy and host your productizer" />
-  <p>
-    Deploy and host your productizer. For the next steps we will need the base URL at which it is
-    responding, for example <em>https://productizer.example.com/</em>.
-  </p>
-  <SectionTitle title="Register your data source in the Developer Portal" />
-  <p>Log in to the Developer Portal.</p>
+  <SectionTitle title="Register your data source in the Dataspace" />
+  <p>Go to the dataspace and click on <em>My groups</em>.</p>
```

Comment:
```suggestion

```

This sentence is now duplicated in the beginning of the first paragraph under the subheading.